Sources/OpenTelemetrySdk/Trace/RecordEventsReadableSpan.swift

```diff
@@ -146,7 +146,7 @@ public class RecordEventsReadableSpan: ReadableSpan {
                                  attributes: AttributesDictionary,
                                  links: [SpanData.Link],
                                  totalRecordedLinks: Int,
-                                 startTime: Date) -> RecordEventsReadableSpan
```

Comment:
should the comment for the `startTime` be updated to indicate the behavior when not set?

Reply:
Yes, its true. I tried to clarify it